src/main/java/website/frontrow/level/Level.java

```diff
@@ -78,7 +78,7 @@ public Level(ArrayList<Player> players, ArrayList<Unit> units, Grid<Cell> cells)
      */
     public void addUnit(Unit unit)
```

Comment:
I only changed the name of the variable, I did not create the method. I will create a test though.